lib/services/new_question_service.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:project_eureka_flutter/models/question_model.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart' as DotEnv;
+import 'dart:async';
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+class NewQuestionService {
+  void postNewQuestion(QuestionModel question) async {
+    await DotEnv.load();
+
+    final response = await http.post(
```

Comment:
This should return a `Future<http.Response>` that will return that response on line 11. You can take a look at how this is implemented inside profile_onboarding_service.dart.

Doing this will prevent this error while waiting for the data to be sent: 
![image](https://user-images.githubusercontent.com/36316432/113464725-d710e500-93e3-11eb-95b8-19e916ab011a.png)

Reply:
Done